sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarTSCoreBundle.java

```diff
@@ -107,6 +112,11 @@ public String getNodeExecutable() {
     return configuration.get(TypeScriptPlugin.NODE_EXECUTABLE).orElse(TypeScriptPlugin.NODE_EXECUTABLE_DEFAULT);
   }
 
+  @Override
+  public List<String> getSonarTSServerCommand() {
+    return Arrays.asList(configuration.get(TypeScriptPlugin.NODE_EXECUTABLE).get(), this.sonartsServer.getAbsolutePath());
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=sonarlint-integration-poc&issues=AWDacfOsmexALlUbKFRV&open=AWDacfOsmexALlUbKFRV)

[ISSUE_KEY]: AWDacfOsmexALlUbKFRV